tutorials/install-cockroachdb-scaleway-instances/index.mdx

```diff
@@ -0,0 +1,480 @@
+---
+meta:
+  title: Installing CockroachDB on Scaleway Instances
+  description: This page shows you how to deploy a secure multi-node CockroachDB cluster on Scaleway, using the Scaleway managed load balancing service to distribute client traffic.
+content:
+  h1: Installing CockroachDB on Scaleway Instances
+  paragraph: This page shows you how to deploy a secure multi-node CockroachDB cluster on Scaleway, using the Scaleway managed load balancing service to distribute client traffic.
+tags:  cockroachdb instances sql database
+categories:
+  - compute
+  - instances
+dates:
+  validation: 2023-31-08
+  posted: 2023-31-08
+---
+
+[CockroachDB](https://www.cockroachlabs.com/docs/v23.1/frequently-asked-questions) is an open-source, distributed SQL database designed to be scalable, reliable and consistent.
+
+CockroachDB scales horizontally. This means that in case of increased data volume, you can simply add more servers to your CockroachDB cluster instead of scaling vertically by adding more resources (memory, CPU or storage) to a single server.
```

Comment:
```suggestion
CockroachDB scales horizontally. This means that in case of increased data volume, you can add more servers to your CockroachDB cluster instead of scaling vertically by adding more resources (memory, CPU, or storage) to a single server.
```